charts/hivemq-operator/Chart.yaml

```diff
@@ -10,7 +10,7 @@ keywords:
   - messaging
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.11.6
+version: 0.11.7
```

Comment:
Shouldn't this be 0.12.0 then because it adds new functionality?

Reply:
good point, thanks